certified-connectors/PowerAssist/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2539 @@
+              "type": "object",
+              "properties": {
+                "email": {
+                  "type": "string",
+                  "description": "email",
+                  "title": "String"
+                }
+              },
+              "required": [
+                "email"
+              ]
+            }
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ]
+      }
+    },
+    "/validate/regex": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Result": {
+                  "type": "boolean",
+                  "description": "Result"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "401 Unauthorized - Please check that you have the correct RapidAPI Key and RapidAPI subscription level to use this action."
+          },
+          "404": {
+            "description": "404 Not Found - if this issue persists, reach out to powerassistsupport@elevate-digital.com."
+          },
+          "500": {
+            "description": "500 Internal Server Error - if this issue persists, reach out to powerassistsupport@elevate-digital.com."
+          }
+        },
+        "summary": "Validate a String based on RegEx",
+        "description": "Validates that a String against a supplied RegEx pattern. Include the leading and trailing '/' in your RegEx pattern and optionally append flags.",
+        "operationId": "ValidateRegex",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "string": {
+                  "type": "string",
+                  "description": "string",
+                  "title": "String"
+                },
+                "pattern": {
+                  "type": "string",
+                  "description": "pattern",
+                  "title": "RegEx Pattern"
+                }
+              },
+              "required": [
+                "pattern",
+                "string"
+              ]
+            }
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ]
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "API Key": {
+      "type": "apiKey",
+      "in": "header",
+      "name": "x-rapidapi-key"
+    }
+  },
+  "security": [
+    {
+      "API Key": []
+    }
+  ],
+  "tags": [],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://powerautomateassist.com"
```

Comment:
Hi @sriyen-msft I've updated the website at that URL, can you please try again now?

Reply:
Hi @sriyen-msft please let me know when you are able to check again.

Reply:
@dwmorton18 I can not see the update that happened to your swagger in the PR, If you have updated the Website URL in your local repo, please push it to the remote branch, so that we can all see the change.